pennylane/plugins/default_qubit.py

```diff
@@ -370,9 +370,18 @@ def expval(self, observable, wires, par):
 
         return ev
 
-    def var(self, expectation, wires, par):
-        # measurement/expectation value <psi|A|psi>
-        A = self._get_operator_matrix(expectation, par)
+    def var(self, observable, wires, par):
+        r"""Variance of observable on specified wires.
+
+        Args:
+          observable      (str): name of the observable
+          wires (Sequence[int]): target subsystems
+          par    (tuple[float]): parameter values
+
+        Returns:
+          float: variance :math:`\mathrm{var}(A) = \bra{\psi}A^2\ket{\psi} - \bra{\psi}A\ket{\psi}^2`
+            """
+        A = self._get_operator_matrix(observable, par)
```

Comment:
same as above. The only time an inherited method docstring needs to be written is if the functionality/arguments change, or special notes on implementation need to be included.